src/Zicht/Bundle/AdminBundle/Security/Handler/RoleSecurityHandler.php

```diff
@@ -20,17 +20,8 @@ class RoleSecurityHandler extends BaseRoleSecurityHandler
      */
     public function isGranted(AdminInterface $admin, $attributes, $object = null)
     {
-        // sonata-admin 2
-        if (isset($this->securityContext)) {
-            if ($this->securityContext->isGranted($attributes, $object)) {
-                return true;
-            }
-        }
-        // sonata admin 3
-        if (isset($this->authorizationChecker)) {
-            if ($this->authorizationChecker->isGranted($attributes, $object)) {
-                return true;
-            }
+        if ($this->authorizationChecker->isGranted($attributes, $object)) {
```

Comment:
i think you still got dependency on old version in composer file.

Reply:
Fixed by dropping support for v2 of doctrine-orm-admin-bundle